examples/demo-app/src/cloud-providers/google-provider.js

```diff
@@ -0,0 +1,142 @@
+/* global window, gapi */
+import {loadScript} from '../utils/load-script';
+
+// We load `gapi` at run-time from this URL
+const GOOGLE_API_URL = 'https://apis.google.com/js/api.js';
+
+export default class GoogleProvider {
+  // Create a new Google Account provider
+  // To access Google Drive, specify it in scopes
+  // @param scopes = ['https://www.googleapis.com/auth/drive'];
+  constructor({clientId, appKey, name, appName, icon, discoveryDocs, scopes = []}) {
+    // All cloud-providers providers must implement the following properties
```

Comment:
I’m not clear what the hierarchy is here. My understanding is the following:
GoogleDroveProvider and DropboxProvider are peers and GoogleProvider and DropboxSdk are their respective parents. 

But constructors don’t reflect such hierarchy, why google provider have an icon given it is an api layer? 

If my first assumptions is wrong, can you clarify in the PR description the role of GoolgeProvider and GoogleCloudProvider?

I only see google provider in it